devicemaps.json

```
@@ -97,7 +97,8 @@
           }
         ]
       },
-      "90500-0151-G53": {
+      "90500-0151": {
```

@haberkornsam small nit but 

```suggestion
      "90500-0151-": {
```

Just to make a bit more explicity